README.md

```diff
@@ -251,39 +251,49 @@ Using the [Portal](http://portal.ditto.live) you can create apps that sync to th
 
 ```tsx
 /** Example of a React root component setting up a single ditto instance that uses a development connection */
+import { useEffect } from 'react'
+
 const RootComponent = () => {
-  const { create, getAuthenticationRequired, getTokenExpiresInSeconds, authenticate } = useOnlineIdentity()
-  
+  const { create, getAuthenticationRequired, getTokenExpiresInSeconds } = useOnlineIdentity()
```

Comment:
`getAuthenticationRequired` and `getTokenExpiresInSeconds` aren't used in this code sample nor the one above. Should we remove them?

Reply:
I left them in the documentation simply to show that they're returned by the hook, as part of the documentation.

Reply:
Ah, makes sense.